packages/react-charts/build-single-packages.json

```diff
@@ -0,0 +1,17 @@
+{
+    "packageName": "@patternfly/react-charts",
+    "modules": [ 
+        {
+            "module": "dist/esm",
+            "moduleType": "esm"
+        },
+        {
+            "module": "dist/js",
+            "moduleType": "cjs"
+        }
+    ],
+    "blackList": [
+        "dist/esm/deprecated/index.js",
+        "dist/esm/next/index.js"
+    ]
+}
```

Comment:
I'm a bit confused as to what the purpose of the blacklist is?

Reply:
It's to remove certain barrel files you don't want included in the build.